backend/src/hatchling/licenses/parse.py

```diff
@@ -1,7 +1,13 @@
+from __future__ import annotations
+
+from typing import Dict, Union, cast
+
 from hatchling.licenses.supported import EXCEPTIONS, LICENSES
 
+LicenseType = Dict[str, Union[str, bool]]
```

Comment:
maybe? https://docs.python.org/3/library/typing.html#typing.TypedDict

Reply:
As I thought, the future wouldn't bring this to 3.7 ... Should I revert to the previous solution or any other ideas?

Reply:
revert, but no need for constant